### PR TITLE
enh(mc): update Azure AKS pack

### PR DESCRIPTION
**Pull request by omercier**
## Description

update Azure AKS pack

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md [hunk 1]
```diff
@@ -5,316 +5,724 @@ title: Azure Kubernetes Service
...
 import TabItem from '@theme/TabItem';
 
-## Vue d'ensemble
-
-Azure Kubernetes Service (AKS) simplifie le déploiement d’un cluster Kubernetes managé en déléguant la charge opérationnelle à Azure. En tant que service Kubernetes hébergé, Azure gère des tâches critiques telles que l’analyse de l’intégrité et la maintenance.
-
-Le connecteur de supervision Centreon *Azure Kubernetes Service* peut s'appuyer sur l'API d'Azure ou bien Azure CLI pour collecter les métriques relatives à AKS.
-
-## Contenu du Pack
+## Contenu du pack
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:8]: do we not keep the first section?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md [hunk 2]
```diff
@@ -5,316 +5,724 @@ title: Azure Kubernetes Service
...
-| Traffic               | Cloud-Azure-Compute-Aks-Traffic-Api               | X      |
-| Unneeded-nodes        | Cloud-Azure-Compute-Aks-Unneeded-Nodes-Api        | X      |
+<Tabs groupId="sync">
+<TabItem value="Cloud-Azure-Compute-Aks-custom" label="Cloud-Azure-Compute-Aks-custom">
+
+| Alias                 | Modèle de service                                        | Description                                                                     |
+|:----------------------|:---------------------------------------------------------|:--------------------------------------------------------------------------------|
+| Allocatable-resources | Cloud-Azure-Compute-Aks-Allocatable-Resources-Api-custom | Contrôle de la mémoire et des coeurs CPU allouables restants sur le Cluster AKS |
+| Cpu-Usage             | Cloud-Azure-Compute-Aks-Cpu-Usage-Api-custom             | Contrôle le CPU du cluster AKS                                                  |
+| Health                | Cloud-Azure-Compute-Aks-Health-Api-custom                | Contrôle le statut du Cluster AKS                                               |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:26]:
```suggestion
| Health                | Cloud-Azure-Compute-Aks-Health-Api-custom                | Contrôle le statut du cluster AKS                                               |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md [hunk 3]
```diff
@@ -5,316 +5,724 @@ title: Azure Kubernetes Service
...
+| Memory                | Cloud-Azure-Compute-Aks-Memory-Api-custom                | Contrôle le taux d'utilisation mémoire du Cluster AKS                           |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:27]:
```suggestion
| Memory                | Cloud-Azure-Compute-Aks-Memory-Api-custom                | Contrôle le taux d'utilisation mémoire du cluster AKS                           |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md [hunk 4]
```diff
@@ -5,316 +5,724 @@ title: Azure Kubernetes Service
...
+| Node-State            | Cloud-Azure-Compute-Aks-Node-State-Api-custom            | Contrôle l'état des noeuds du cluster AKS                                       |
+| Pod-State             | Cloud-Azure-Compute-Aks-Pod-State-Api-custom             | Contrôle l'état des Pods du cluster AKS                                         |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:29]:
```suggestion
| Pod-State             | Cloud-Azure-Compute-Aks-Pod-State-Api-custom             | Contrôle l'état des pods du cluster AKS                                         |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md [hunk 5]
```diff
@@ -5,316 +5,724 @@ title: Azure Kubernetes Service
...
+| Storage               | Cloud-Azure-Compute-Aks-Storage-Api-custom               | Contrôle les statistiques de stockage du Cluster AKS                            |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:30]:
```suggestion
| Storage               | Cloud-Azure-Compute-Aks-Storage-Api-custom               | Contrôle les statistiques de stockage du cluster AKS                            |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md [hunk 6]
```diff
@@ -5,316 +5,724 @@ title: Azure Kubernetes Service
...
+| Traffic               | Cloud-Azure-Compute-Aks-Traffic-Api-custom               | Contrôle l'utilisation du réseau du Cluster AKS                                 |
+| Unneeded-nodes        | Cloud-Azure-Compute-Aks-Unneeded-Nodes-Api-custom        | Contrôle du nombre de noeuds inutiles sur le Cluster AKS                        |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:32]:
```suggestion
| Unneeded-nodes        | Cloud-Azure-Compute-Aks-Unneeded-Nodes-Api-custom        | Contrôle du nombre de noeuds inutiles sur le cluster AKS                        |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md [hunk 7]
```diff
@@ -5,323 +5,722 @@ title: Azure Kubernetes Service
...
-| Health                | Cloud-Azure-Compute-Aks-Health-Api                | X       |
-| Storage               | Cloud-Azure-Compute-Aks-Storage-Api               | X       |
-| Traffic               | Cloud-Azure-Compute-Aks-Traffic-Api               | X       |
-| Unneeded-nodes        | Cloud-Azure-Compute-Aks-Unneeded-Nodes-Api        | X       |
+<Tabs groupId="sync">
+<TabItem value="Cloud-Azure-Compute-Aks-custom" label="Cloud-Azure-Compute-Aks-custom">
+
+| Service Alias         | Service Template                                         | Service Description                                            |
+|:----------------------|:---------------------------------------------------------|:---------------------------------------------------------------|
+| Allocatable-resources | Cloud-Azure-Compute-Aks-Allocatable-Resources-Api-custom | Check memory and CPU cores allocatable for AKS Cluster         |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:23]:
```suggestion
| Allocatable-resources | Cloud-Azure-Compute-Aks-Allocatable-Resources-Api-custom | Check memory and CPU cores allocatable for AKS cluster         |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md [hunk 8]
```diff
@@ -5,323 +5,722 @@ title: Azure Kubernetes Service
...
+| Cpu-Usage             | Cloud-Azure-Compute-Aks-Cpu-Usage-Api-custom             | Monitor the CPU usage of the AKS cluster                       |
+| Health                | Cloud-Azure-Compute-Aks-Health-Api-custom                | Check AKS Cluster state                                        |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:25]:
```suggestion
| Health                | Cloud-Azure-Compute-Aks-Health-Api-custom                | Check AKS cluster state                                        |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md [hunk 9]
```diff
@@ -5,323 +5,722 @@ title: Azure Kubernetes Service
...
+| Memory                | Cloud-Azure-Compute-Aks-Memory-Api-custom                | Monitors the memory usage of the AKS Cluster                   |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:26]:
```suggestion
| Memory                | Cloud-Azure-Compute-Aks-Memory-Api-custom                | Monitors the memory usage of the AKS cluster                   |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md [hunk 10]
```diff
@@ -5,323 +5,722 @@ title: Azure Kubernetes Service
...
+| Node-State            | Cloud-Azure-Compute-Aks-Node-State-Api-custom            | Monitors the state of the AKS cluster nodes                    |
+| Pod-State             | Cloud-Azure-Compute-Aks-Pod-State-Api-custom             | Monitors the state of the AKS cluster Pods                     |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:28]:
```suggestion
| Pod-State             | Cloud-Azure-Compute-Aks-Pod-State-Api-custom             | Monitors the state of the AKS cluster pods                     |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md [hunk 11]
```diff
@@ -5,323 +5,722 @@ title: Azure Kubernetes Service
...
+| Storage               | Cloud-Azure-Compute-Aks-Storage-Api-custom               | Check AKS Cluster storage                                      |
+| Traffic               | Cloud-Azure-Compute-Aks-Traffic-Api-custom               | Check AKS Cluster network usage                                |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:30]:
```suggestion
| Traffic               | Cloud-Azure-Compute-Aks-Traffic-Api-custom               | Check AKS cluster network usage                                |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md [hunk 12]
```diff
@@ -5,323 +5,722 @@ title: Azure Kubernetes Service
...
+| Unneeded-nodes        | Cloud-Azure-Compute-Aks-Unneeded-Nodes-Api-custom        | Check amount of unneeded nodes on AKS Cluster                  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:31]:
```suggestion
| Unneeded-nodes        | Cloud-Azure-Compute-Aks-Unneeded-Nodes-Api-custom        | Check amount of unneeded nodes on AKS cluster                  |
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md [hunk 13]
```diff
@@ -5,323 +5,722 @@ title: Azure Kubernetes Service
...
+| Unschedulable-Pods    | Cloud-Azure-Compute-Aks-Unschedulable-Pods-Api-custom    | Monitors the presence of unschedulable Pods on the AKS cluster |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:32]:
```suggestion
| Unschedulable-Pods    | Cloud-Azure-Compute-Aks-Unschedulable-Pods-Api-custom    | Monitors the presence of unschedulable pods on the AKS cluster |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md (hunk 1 above)
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:8]:
```suggestion
Azure Kubernetes Service (AKS) simplifie le déploiement d’un cluster Kubernetes managé en déléguant la charge opérationnelle à Azure. En tant que service Kubernetes hébergé, Azure gère des tâches critiques telles que l’analyse de l’intégrité et la maintenance.

Le connecteur de supervision Centreon *Azure Kubernetes Service* peut s'appuyer sur l'API d'Azure ou bien Azure CLI pour collecter les métriques relatives à AKS.

## Contenu du pack
```

File: pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md [hunk 14]
```diff
@@ -5,323 +5,722 @@ title: Azure Kubernetes Service
...
 import TabItem from '@theme/TabItem';
 
-## Overview
-
-Azure Kubernetes Service (AKS) simplifies deploying a managed Kubernetes cluster in Azure by offloading the operational overhead to Azure. As a hosted Kubernetes service, Azure handles critical tasks, like health monitoring and maintenance.
-
-The Centreon Plugin *Azure Kubernetes Service* can rely on Azure API or Azure CLI to collect the metrics related to the *AKS* service.
-
-## Pack Assets
+## Pack assets
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/cloud-azure-compute-aks.md:8]:
```suggestion
Azure Kubernetes Service (AKS) simplifies deploying a managed Kubernetes cluster in Azure by offloading the operational overhead to Azure. As a hosted Kubernetes service, Azure handles critical tasks, like health monitoring and maintenance.

The Centreon Plugin *Azure Kubernetes Service* can rely on Azure API or Azure CLI to collect the metrics related to the *AKS* service.

## Pack assets
```

**Pull request merged**